lib/ansible.js

```diff
@@ -24,10 +24,18 @@ AbstractAnsibleCommand.prototype.exec = function(options) {
 
   var deferred = Q.defer();
   var processOptions = {};
-  if (options && options.cwd) {
+  if (!options) options = {};
+
+  if (options.cwd) {
     processOptions.cwd = options.cwd;
   }
 
+  // Make Ansible output unbuffered so the stdout and stderr `data` events 
+  // pickup output up more regularly
+  var unbuffered = (options.buffered) ? '' : 1;
+  processOptions.env = process.env;
```

Comment:
Yeah. Your code just made we think that this has been a mistake on my part, and in fact we need to reset the environment. Not critical though for this PR.